common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -996,69 +989,142 @@ div.problem {
     border: 1px solid $gray-l3;
   }
 
+  .message {
+    font-size: inherit;
+  }
+
   .detailed-solution {
-    > p:first-child {
-      @extend %t-strong;
-      color: $gray;
-      text-transform: uppercase;
-      font-style: normal;
-      font-size: 0.9em;
-    }
+    > p {
+      margin: 0;
 
-    p:last-child {
-      margin-bottom: 0;
+      &:first-child {
+        @extend %t-strong;
+        margin-bottom: 0;
+      }
     }
   }
 
-  .detailed-targeted-feedback {
-    > p:first-child {
-       @extend %t-strong;
-      color: $incorrect;
-      text-transform: uppercase;
-      font-style: normal;
-      font-size: 0.9em;
-    }
+  .detailed-targeted-feedback,
+  .detailed-targeted-feedback-partially-correct,
+  .detailed-targeted-feedback-correct {
+    > p {
+        margin: 0;
+        font-weight: normal;
 
-    p:last-child {
-      margin-bottom: 0;
+        &:first-child {
+          @extend %t-strong;
+        }
     }
   }
 
-  .detailed-targeted-feedback-partially-correct {
-    > p:first-child {
-       @extend %t-strong;
-      color: $partiallycorrect;
-      text-transform: uppercase;
-      font-style: normal;
-      font-size: 0.9em;
+  div.capa_alert {
+    margin-top: $baseline;
+    padding: 8px 12px;
+    border: 1px solid #ebe8bf;
+    border-radius: 3px;
+    background: #fffcdd;
+    font-size: 0.9em;
+  }
+
+  .notification {
+    margin-top: $baseline / 2;
+    padding: ($baseline * 0.4) ($baseline / 2) ($baseline / 5);
```

Comment:
RTL, also this padding rule only has three arguments - it has to be either 1, 2, or 4